devel/qbe/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                qbe
+version             1.0
+categories          devel
+license             MIT
+maintainers         {@alterae} \
+                    openmaintainer
+description         Compiler backend
+long_description    QBE is a compiler backend that aims to provide 70% of the performance of industrial optimizing compilers in 10% of the code.
+homepage            https://c9x.me/compile/
+master_sites        https://c9x.me/compile/release/
+distfiles           qbe-${version}.tar.xz
+
+checksums           rmd160  984f7c1658c4c6f3302e40c61540dbc409ae1167 \
+                    sha256  257ef3727c462795f8e599771f18272b772beb854aacab97e0fda70c13745e0c \
+                    size    237756
+
+use_configure       no
```

Comment:
without a `configure` command it's up to the Portfile author to make sure the port will be [UsingTheRightCompiler](https://trac.macports.org/wiki/UsingTheRightCompiler) and associated MacPorts' flags. Oftentimes the `makefile` PortGroup can help/automate this. 

Admittedly, I haven't looked into the specifics of this package but please take a look at the above-mentioned link to make sure the correct stuff is happening.